src/index.js

```diff
@@ -86,33 +109,11 @@ const loadMargaritaInstruction = (userQuery) => {
         drinkType.addEventListener("click", () => {
           goToDDP(drinkType, drink)
         })
-
-
-        regresar.addEventListener("click", () => {
-          // regresar a vista original (lista de bebidas)
-          document.querySelector(".flex-container").style.display = "flex"
-          document.querySelector(".busqueda").style.display = "flex"
-          regresar.style.display = "none"
-          document.querySelector(".class-old-image").style.display = "none"
-          document.querySelector(".parrafo-instrucciones").style.display = "none"
-          document.querySelector(".borrar-ingredientes").style.display = "none"
-          titulo.innerText = "Pollos BAR"
-          instructionsDiv.style.display = "none"
-          ingredientsDiv.style.display = "none"
-          flexContainer.style.display = "none"
-
-          const oldParagraphs = document.getElementsByClassName("drink-type")
-
-          for (let index = oldParagraphs.length - 1; index >= 0; index--) {
-            oldParagraphs[index].remove()
-
-          }
-
-
+        drinkType.addEventListener("click", () => {
```

Comment:
probably you wanted to add this click event to `regresar` insead of drinkType

Reply:
yes. your rigth, now it works